app/assets/javascripts/kaui/kaui.js

```diff
@@ -261,3 +268,57 @@ function isBlank(value) {
         return false;
     }
 }
+
+// this function set popover for all tags that have class object-id-popover
+// attributes:
+//      data-id = content of the popover,object id; required
+//      title = title of the popover; not required
+//      id = (must be {{id}}-popover) used to close popover when the copy image is clicked; if present; if not present a timeout of 5s will apply; not required
+function setObjectIdPopover(){
+    $(".object-id-popover").each(function(idx, e){
+        $(this).popover('destroy');
+        $(this).off("shown.bs.popover");
+
+        $(this).popover({
+            html: true,
+            content: function() {
+                var template = '<div class="{{id}}-content" >' +
+                    '{{id}}&emsp;<i id="{{id}}-copy" class="fa fa-clipboard" aria-hidden="true"></i> ' +
+                    '<input id="{{id}}-placeholder" class="form-control hidden"> ' +
+                    '</div>';
+
+                var popover_html = Mustache.render( template , { id: $(this).data("id") });
+                return popover_html;
+            },
+            trigger: 'hover',
+            delay: { "show": 100, "hide": 4000 }
+        });
+
+        $(this).on("shown.bs.popover", function(e) {
+            var objectId = $(this).data('id');
+            var copyIdImg = $("#" + objectId + "-copy");
+
+            copyIdImg.data("popover",$(this).attr("id"));
+            copyIdImg.click(function(e){
+                var id = ($(this).attr("id")).replace('-copy','');
+                var placeholder = $("#" + objectId + "-placeholder");
+                var popover = $("#" + copyIdImg.data("popover"));
+                placeholder.val(id);
+                placeholder.removeClass("hidden");
+                placeholder.select();
+
+                document.execCommand("Copy");
+                placeholder.addClass("hidden");
+                ajaxAlert("Id [" + id + "] was copied into the clipboard!", 4000);
```

Comment:
It would be nice to also change the color (e.g. `alert-info`), since it's not an error case.